app/BUILD.bazel

```diff
@@ -258,16 +349,55 @@ in order to keep Gradle building.
  )
 for file in ALTERED_VIEWS]
 
+# Binding Adapters
 
 # TODO(#1617): Remove genrules post-gradle
-# Adapter files altered by genrules to be excluded in the app library.
+'''
+Binding adapter files under the org/oppia/app/databinding folder that need to be altered by the
+databinding binding adapter genrule.
+
+Please note that binding adapters have been split up into two types based on their location:
+databinding_adapters can be found under org/oppia/app/databinding and recyclerview_adapters can be
+found under org/oppia/app/recyclerview. Note that a genrule has not been created for
+recyclerview_adapters as it is not currently needed.
+
+Place your file here if:
+ - It is a binding adapter
+ - It imports resources (ex: `import org.oppia.app.R`)
+ - It is filed under org/oppia/app/databinding
```

Comment:
What about binding adapters that don't fit one of the two above requirements?

Reply:
I decided separating them was unnecessary - they are in one library now